### PR TITLE
add changelog entries for 16.10.5 release.

### PR DESCRIPTION
**Pull request by patflynn**

### CODE REVIEW

**Comment by patflynn** [comment]
@joaoandremartins 
@aslo

File: google-cloud-tools-plugin/resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -37,9 +37,28 @@
...
+      development server with the latest Java 8 SDK. Please run 'gcloud components update' from your
+      shell to ensure you have the latest Cloud SDK release.
+      </p>
+
       <h2>[16.10.5]</h2>
+      <h3>Fixed</h3>
       <ul>
-        <li>Fixed issue with local development server debug mode when changes are made while the server is running. (<a href="https://github.com/GoogleCloudPlatform/google-cloud-intellij/issues/972">#972</a>)</li>
+        <li>
+        Fixed issue with local development server debug mode when changes are made while the server is
```
Comment (joaoandremartins) [inline google-cloud-tools-plugin/resources/META-INF/plugin.xml]: Just out of curiosity, did we add the newline here? Anything to do with style?

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,8 +4,18 @@ All notable changes to this project will be documented in this file.
...
+shell to ensure you have the latest Cloud SDK release.
+
 ### Fixed
-- Fixed issue with local development server debug mode when changes are made while the server is running. ([#972](https://github.com/GoogleCloudPlatform/google-cloud-intellij/issues/972))
+- Fixed issue with local development server debug mode when changes are made while the server is 
+running. ([#972](https://github.com/GoogleCloudPlatform/google-cloud-intellij/issues/972))
+- Better wording when the development server has an invalid Cloud SDK path. 
+([#1043](https://github.com/GoogleCloudPlatform/google-cloud-intellij/issues/1043)).
+- Update run configuration names to be prefixed with 'Google ..'
+([#1043](https://github.com/GoogleCloudPlatform/google-cloud-intellij/issues/1041)).
```
Comment (etanshaul) [inline CHANGELOG.md]: this link is incorrect - should be 1043, like you have in the label

File: google-cloud-tools-plugin/resources/META-INF/plugin.xml [hunk 3]
```diff
@@ -37,9 +37,28 @@
...
+        running. (<a href="https://github.com/GoogleCloudPlatform/google-cloud-intellij/issues/972">#972</a>)
+        </li>
+        <li>
+        Better wording when the development server has an invalid Cloud SDK path.
+        (<a href="https://github.com/GoogleCloudPlatform/google-cloud-intellij/issues/1043">#1043</a>).
+        </li>
+        <li>
+        Update run configuration names to be prefixed with 'Google ..'
+        (<a href="https://github.com/GoogleCloudPlatform/google-cloud-intellij/issues/1041">#1043</a>).
```
Comment (etanshaul) [inline google-cloud-tools-plugin/resources/META-INF/plugin.xml]: same comment about the link: href should be 1043

File: google-cloud-tools-plugin/resources/META-INF/plugin.xml [hunk 4]
```diff
@@ -57,7 +57,7 @@
         </li>
         <li>
         Update run configuration names to be prefixed with 'Google ..'
-        (<a href="https://github.com/GoogleCloudPlatform/google-cloud-intellij/issues/1041">#1043</a>).
+        (<a href="https://github.com/GoogleCloudPlatform/google-cloud-intellij/issues/1041">#1041</a>).
```
Comment (etanshaul) [inline google-cloud-tools-plugin/resources/META-INF/plugin.xml]: Oops nevermind, 1043 is the PR. 1021 is correct like your last update.

**Comment by etanshaul** [comment]
lgtm

File: google-cloud-tools-plugin/resources/META-INF/plugin.xml (hunk 1 above)
Comment (patflynn) [inline google-cloud-tools-plugin/resources/META-INF/plugin.xml]: eh.. 100 line width? I dunno. no good reason probably.

**Pull request merged**